docs/faq.md

```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
+* A grace period for all users with any number of contributors is ongoing from now until July 31, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier. This updated Enterprise tier includes such features as single-tenancy and a secure network tunnel, ideal for organizations with increased infrastructure, reporting, and other custom requirements.
+
+If the number of contributors has exceeded the usage limit, you will need to purchase licensing for the 11th contributor onward. 
+
+### What is the usage limit?
+
+The usage limit is 10 contributors. A contributor is someone who has made at least one commit to a scanned private repository within the last 30 days.
+
+### How are contributors calculated?
+
+Contributors are calculated using `git log` over the past 30 days.
+
+### What happens when the usage limit is exceeded?
+
+Semgrep scans stop when the usage limit is exceeded. You can resume scanning through the following:
+
+* A one-time 30-day free trial (automatically triggered when the usage limit is exceeded for the first time)
+* By purchasing additional licenses
+* By waiting for the next billing cycle
+
+### What if the first scan exceeds the contributor usage limit?
+
+Semgrep will complete the first scan and a one-time 30-day free trial will automatically start. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### How is the cost calculated for the Team tier?
+
+The 11th contributor and onwards are charged. A 20-member engineering team would pay $7,200 per year for both products at the discounted bundled rate.
+
+For example:
+
+| Product  | List price | Months | Quantity | Subtotal | Discount | Total price |
+| -------  | ------ | ------- | ----- | ------ | ---- | ----- |
+| Semgrep Supply Chain (bundle)  | $30 | 12 | 20 | $7,200 | 10 licenses ($3,600) | $3600 |
+| Code (bundle)  | $30 | 12 | 20 | $7,200 | 10 licenses ($3,600) | $3600 |
+| Total (bundle) |    | | | | | $7200 |
+
+### Can a single product be purchased?
+Yes, you can buy a single product. Products can also be disabled from the Settings page.
+
+### Can I purchase a different number of licenses per product?
+
+If you are purchasing licenses for both products, you can't purchase a different number of licenses per product because the products are bundled together. Both Semgrep Supply Chain and Semgrep Code will run on every scan unless disabled (for all projects) from the settings page.
```

Comment:
@s-santillan, I think the original FAQ was more clear here in that it was more explicit. The answer there was "No" 😅

Reply:
Yeah, have edited this to say "no" with an example.